BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -37,4 +43,65 @@ public async void ApproveCharter_CallsApproveLandlord()
         // Assert
         A.CallTo(() => landlordService.ApproveLandlord(landlord.Id, adminUser)).MustHaveHappened();
     }
+
+    [Fact]
+    public void ViewProperties_CalledByAdmin_CanReturnOtherUsersPropertyList()
+    {
+        // Arrange
+        var adminUser = CreateAdminUser();
+        var fakeLandlordService = A.Fake<ILandlordService>();
+        var fakePropertyService = A.Fake<IPropertyService>();
+        var _underTest =
+            new LandlordController(A.Fake<ILogger<LandlordController>>(), fakeLandlordService,
+                fakePropertyService, null!);
+        MakeUserPrincipalInController(adminUser, _underTest);
+
+        // Act
+        var results = new List<ViewResult?>();
+        A.CallTo(() => fakePropertyService.GetPropertiesByLandlord(1))
+            .Returns(A.CollectionOfFake<PropertyDbModel>(10).ToList());
+        results.Add(_underTest.ViewProperties(1) as ViewResult);
+        A.CallTo(() => fakePropertyService.GetPropertiesByLandlord(2))
+            .Returns(A.CollectionOfFake<PropertyDbModel>(10).ToList());
+        results.Add(_underTest.ViewProperties(2) as ViewResult);
+
+        // Assert
+        foreach (var result in results)
+        {
+            result.Should().BeOfType<ViewResult>();
+            result.Should().NotBeNull();
+            result!.Model.Should().BeOfType<PropertiesDashboardViewModel>();
+            result.Model.As<PropertiesDashboardViewModel>().Properties.Count.Should()
+                .Be(10); //Properties.Count.Should().Be(10);
+        }
+    }
+
+    [Fact]
+    public void ViewProperties_CalledByNonAdmin_CannotReturnOtherUsersPropertyList()
+    {
+        // Arrange
+        var unregisteredUser = CreateUnregisteredUser();
+        var fakeLandlordService = A.Fake<ILandlordService>();
+        var fakePropertyService = A.Fake<IPropertyService>();
+        var underTest =
+            new LandlordController(A.Fake<ILogger<LandlordController>>(), fakeLandlordService, null!, null!);
+        MakeUserPrincipalInController(unregisteredUser, underTest);
+
+        // Act
+        var results = new List<StatusCodeResult?>();
+        A.CallTo(() => fakePropertyService.GetPropertiesByLandlord(1))
+            .Returns(A.CollectionOfFake<PropertyDbModel>(10).ToList());
+        results.Add(underTest.ViewProperties(1) as StatusCodeResult);
+        A.CallTo(() => fakePropertyService.GetPropertiesByLandlord(2))
+            .Returns(A.CollectionOfFake<PropertyDbModel>(10).ToList());
+        results.Add(underTest.ViewProperties(2) as StatusCodeResult);

```

Comment:
Not sure there's much value in having two landlords tested if the assertions are the same